stdlib/LibGit2/src/LibGit2.jl

```diff
@@ -972,12 +972,12 @@ function set_ssl_cert_locations(cert_loc)
           Cint(Consts.SET_SSL_CERT_LOCATIONS), cert_file, cert_dir)
 end
 
-function __init__()
-    @check ccall((:git_libgit2_init, :libgit2), Cint, ())
+function initialize()
```

Comment:
might be worth adding `@noinline` here, since the compiler might see this is small enough to be feasible to inline, (even though that'll make us generate worse code)